python/feathub/feature_tables/tests/test_kafka_source_sink.py

```diff
@@ -65,7 +71,24 @@ def tearDownClass(cls) -> None:
         super().tearDownClass()
         cls.kafka_container.stop()
 
-    def test_kafka_source_sink(self):
+    def test_kafka_source_sink_json(self):
+        self._test_kafka_source_sink("json")
```

Comment:
It will be great to have the test mimic users' experience by connecting the protobuf definition with its usage in the test.

After all, we expect there is an easy way for users to define and use protobuf in FeatHub.